### PR TITLE
update to latest black

### PR DESCRIPTION
**Pull request by gaborbernat**
:+1: more checks and formats

### CODE REVIEW

File: tests/test_pytest_plugins.py [hunk 1]
```diff
@@ -81,7 +79,7 @@ def test_path_parts(self, input, expected):
     def test_on_py_path(self):
         cwd_parts = _path_parts(py.path.local())
         folder_parts = _path_parts(py.path.local("a/b/c"))
-        assert folder_parts[len(cwd_parts):] == ["a", "b", "c"]
+        assert folder_parts[len(cwd_parts) :] == ["a", "b", "c"]
```
Comment (obestwalter) [inline tests/test_pytest_plugins.py]: wat?

File: tox.ini [hunk 2]
```diff
@@ -87,6 +87,7 @@ commands = python -m pip list --format=columns
 [flake8]
 max-complexity = 22
 max-line-length = 99
+ignore = E203, W503
```
Comment (obestwalter) [inline tox.ini]: is PEP8 starting to disagree with black?

File: tests/test_pytest_plugins.py (hunk 1 above)
Comment (RonnyPfannschmidt) [reply tests/test_pytest_plugins.py]: @ambv - why is that?

Comment (RonnyPfannschmidt) [reply tests/test_pytest_plugins.py]: reported upstream

Comment (RonnyPfannschmidt) [reply tests/test_pytest_plugins.py]: https://github.com/PyCQA/pycodestyle/issues/373 - black is right it seems

Comment (gaborbernat) [reply tests/test_pytest_plugins.py]: Just my thoughts about this seem strange at first (like many other black style formats) but it does make sense once you look into it more in depth.

Comment (ambv) [reply tests/test_pytest_plugins.py]: @rpkilby, if you look at the "Yes"/"No" examples below this section, you will see function calls explicitly using a space separating the colon.

File: tox.ini (hunk 2 above)
Comment (ambv) [reply tox.ini]: No, pycodestyle disagrees with PEP 8.

File: tests/test_pytest_plugins.py (hunk 1 above)
Comment (rpkilby) [reply tests/test_pytest_plugins.py]: @ambv the examples use a space, but it doesn't seem to be a part of the recommendation. At least, that's not how the text reads to me.

Regardless, the style black enforces *is* compliant, I just don't happen to like that particular choice. But again, that's just my irrelevant preference.

Comment (obestwalter) [reply tests/test_pytest_plugins.py]: Thanks! I agree that this is what PEP8 describes and I have succesfully been unaware of it for at least a decade, although I have red PEP8 many times. Another big :+1: for @ambv for being the perfectionist he is and getting stuff like this down to the last little detail.

File: tox.ini (hunk 2 above)
Comment (obestwalter) [reply tox.ini]: I tend to believe you here :)

**Pull request merged**